mesonbuild/ast/introspection.py

```diff
@@ -182,7 +182,7 @@ def _add_languages(self, raw_langs: T.List[TYPE_var], required: bool, for_machin
                 if self.subproject:
                     options = {}
                     for k in comp.get_options():
-                        v = copy.copy(self.coredata.options[k])
+                        v = copy.copy(self.coredata.optstore.get_value_object(k))
```

Comment:
I see this pattern 3 times. Why not to have a `get_value_object_copy()` method?

Reply:
Because these will all go away. They will be replaced with named methods that have a proper semantic name. If any copying is to be done it should happen inside the OptionStore object.